examples/7nodes/init.sh

```diff
@@ -28,6 +43,16 @@ while (( "$#" )); do
             consensus=clique
             shift
             ;;
+        --numNodes)
```

Comment:
I'd suggest also documenting '--numNodes' in usage().

Reply:
Updated usage.